Formula/php-cs-fixer@2.rb

```diff
@@ -0,0 +1,26 @@
+class PhpCsFixerAT2 < Formula
+  desc "Tool to automatically fix PHP coding standards issues"
+  homepage "https://cs.symfony.com/"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.19.0/php-cs-fixer.phar"
+  sha256 "dcaf7556479a1cfc7ece11a203c4247d61fabe86e509b0e29539111f6b82018c"
+  license "MIT"
+
+  keg_only :versioned_formula
+  depends_on "php" if MacOS.version <= :el_capitan
```

Comment:
El Capitan is not supported by Homebrew anymore; I would delete this line since we can't test it on our CI

```suggestion
```

Reply:
Actually it should be this AFAIK
```
  uses_from_macos "php", since: :el_capitan
```

Reply:
I just copied it from the latest 2.x release, which isn't that old. Was it wrong all this time?

Reply:
It isn't wrong, we just found a better way to represent a macOS dependency (and don't have the capacity to retroactively fix everything)

Reply:
>   uses_from_macos "php", since: :el_capitan

I think for _new formulae_ we should try to avoid using unsupported (by homebrew) macOS versions (basically because we don't test). Maybe worth adding an audit for this

Reply:
I'm fine with that, I just don't think that's the current practice.